sdk/eventhub/event-hubs/test/client.spec.ts

```diff
@@ -292,7 +292,11 @@ describe("User Agent on EventHubClient on #RunnableInBrowser", function(): void
     properties!["user-agent"].should.startWith(`azsdk-js-azureeventhubs/${packageVersion}`);
     should.equal(properties!.product, "MSJSClient");
     should.equal(properties!.version, packageVersion);
-    should.equal(properties!.framework, `Node/${process.version}`);
+    if (isNode) {
+      should.equal(properties!.framework, `Node/${process.version}`);
+    } else {
+      should.equal(properties!.framework.startsWith("Browser/"), true);
```

Comment:
I believe tests are run in browser only if they are tagged with #RunInBrowser or something like that

Reply:
This test is already being run in the browser because the describe block has that tag. (That’s how I knew to fix this :-) )

Reply:
Woohoo